.changes/bundler-rpm.md

```diff
@@ -0,0 +1,7 @@
+---
+"tauri-bundler": 'patch:enhance'
+"tauri-cli": 'patch:enhance'
+"@tauri-apps/cli": 'patch:enhance'
```

Comment:
and they also have to be minor not patch. They were only patch in v2 because v2 is still in prerelease.

Reply:
Sorry I'm not totally clear what to put here, the CI says I did it wrong. Should it be 'minor:feat'?

Reply:
yep, minor:feat should work